app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(string: str) -> None:
+    string = string.split(" ")
+    command = string[0]
+    file_name = string[1]
+    new_file_name = string[2]
```

Comment:
You can give values for these variables in one line, by using construction "a, b = "a b".split()"